src/containers/EditorContainer.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { useEffect, useState, useCallback } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import EditorNavComponent from 'components/EditorNavComponent';
+import EditorPadComponent from 'components/EditorPadComponent';
+import { Container } from 'core-components';
+import {
+  fetchLanguages,
+  setLanguageSelected,
+  setCode,
+} from 'actions/languageAction';
+import isEmpty from 'utils/isEmpty';
+
+function EditorContainer() {
+  const dispatch = useDispatch();
+
+  const [isDropDownOpen, setDropDownOpen] = useState(false);
+  const { languages, languageSelected, code } = useSelector(
+    (state) => state.languageReducer,
+  );
+
+  useEffect(() => {
+    dispatch(fetchLanguages());
+  }, [dispatch]);
+
+  const handleToggle = useCallback(() => setDropDownOpen(!isDropDownOpen), [
+    isDropDownOpen,
+  ]);
+
+  const handleCode = useCallback(
+    (value) => {
+      dispatch(setCode(value));
+    },
+    [dispatch],
+  );
+  const handleClick = useCallback(
+    (e) => {
+      const langObj = {
+        id: e.currentTarget.getAttribute('id'),
+        name: e.currentTarget.textContent,
+      };
+
+      dispatch(setLanguageSelected(langObj));
+    },
+    [dispatch],
+  );
+
+  let lang = isEmpty(languageSelected)
+    ? ''
+    : languageSelected.name.split(' ')[0].toLowerCase();
+
+  if (lang === 'c++') lang = 'cpp';
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/joshsoftware/OnCoT-Frontend/pullRequest?prid=7073195)
```suggestion
  if (lang === 'c++') {lang = 'cpp';}
```